text/0000-compact-slates.md

```diff
@@ -0,0 +1,499 @@
+        }
+      ],
+      "outputs": [
+        {
+          "features": "Plain",
+          "commit": "0812276cc788e6870612296d926cba9f0e7b9810670710b5a6e6f1ba006d395774",
+          "proof": "dcff6175390c602bfa92c2ffd1a9b2d84dcc9ea941f6f317bdd0f875244ef23e696fd17c71df79760ce5ce1a96aab1d15dd057358dc835e972febeb86d50ccec0dad7cfe0246d742eb753cf7b88c045d15bc7123f8cf7155647ccf663fca92a83c9a65d0ed756ea7ebffd2cac90c380a102ed9caaa355d175ed0bf58d3ac2f5e909d6c447dfc6b605e04925c2b17c33ebd1908c965a5541ea5d2ed45a0958e6402f89d7a56df1992e036d836e74017e73ccad5cb3a82b8e139e309792a31b15f3ffd72ed033253428c156c2b9799458a25c1da65b719780a22de7fe7f437ae2fccd22cf7ea357ab5aa66a5ef7d71fb0dc64aa0b5761f68278062bb39bb296c787e4cabc5e2a2933a416ce1c9a9696160386449c437e9120f7bb26e5b0e74d1f2e7d5bcd7aafb2a92b87d1548f1f911fb06af7bd6cc13cee29f7c9cb79021aed18186272af0e9d189ec107c81a8a3aeb4782b0d950e4881aa51b776bb6844b25bce97035b48a9bdb2aea3608687bcdd479d4fa998b5a839ff88558e4a29dff0ed13b55900abb5d439b70793d902ae9ad34587b18c919f6b875c91d14deeb1c373f5e76570d59a6549758f655f1128a54f162dfe8868e1587028e26ad91e528c5ae7ee9335fa58fb59022b5de29d80f0764a9917390d46db899acc6a5b416e25ecc9dccb7153646addcc81cadb5f0078febc7e05d7735aba494f39ef05697bbcc9b47b2ccc79595d75fc13c80678b5e237edce58d731f34c05b1ddcaa649acf2d865bbbc3ceda10508bcdd29d0496744644bf1c3516f6687dfeef5649c7dff90627d642739a59d91a8d1d0c4dc55d74a949e1074427664b467992c9e0f7d3af9d6ea79513e8946ddc0d356bac49878e64e6a95b0a30214214faf2ce317fa622ff3266b32a816e10a18e6d789a5da1f23e67b4f970a68a7bcd9e18825ee274b0483896a40"
+        }
+      ],
+      "kernels": [
+        {
+          "features": "Plain",
+          "fee": "7000000",
+          "lock_height": "0",
+          "excess": "000000000000000000000000000000000000000000000000000000000000000000",
+          "excess_sig": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+        }
+      ]
+    }
+  },
+  "amount": "60000000000",
+  "fee": "7000000",
+  "height": "5",
+  "lock_height": "0",
+  "ttl_cutoff_height": null,
+  "payment_proof": null,
+  "participant_data": [
+    {
+      "id": "0",
+      "public_blind_excess": "033ac2158fa0077f087de60c19d8e431753baa5b63b6e1477f05a2a6e7190d4592",
+      "public_nonce": "031b84c5567b126440995d3ed5aaba0565d71e1834604819ff9c17f5e9d5dd078f",
+      "part_sig": null,
+      "message": null,
+      "message_sig": null
+    }
+  ]
+}
+```
+
+In version 4 of the Slate format, the initial Slate now looks like the following:
+
+```
+{
+  "ver": "4:3",
+  "sta": "S1",
+  "id": "mavTAjNm4NVztDwh4gdSrQ",
+  "amt": "1000000000",
+  "fee": "8000000",
+  "sigs": [
+    {
+      "xs": "A5sKZUTyGG1FogzBtH8ZvGaVVOFFxusytVLN0rdpX2DE",
+      "nonce": "A1MbUjB5g4fjZ3fTYATVBA8TADHGG77D8gHpzbQTl/IE"
+    }
+  ]
+}
+```
+
+While a "return" slate might look something like the following:
+
+```
+{
+  "ver": "4:3",
+  "id": "7haE++9ZScKLOqM2jvRsjA==",
+  "sta": "S2",
+  "sigs": [
```

Comment:
I think we want to add new `offset` field here before `sigs` begins.

Reply:
:+1: